.zsh/aliases.zsh

```diff
@@ -28,3 +28,4 @@ alias remove_dangling_images="docker rmi \$(docker images -f "dangling=true" -q)
 
 # General aliases
 alias -g rocker_auth="--auth kalbasit:\$(lpass show --password 4984935876)"
+alias ll="ls -al"
```

Comment:
This section here is for the general aliases (notice the `-g`). General aliases can be used anywhere in a command. Please move this alias to the previous paragraph.